README.md

```diff
@@ -10,32 +10,275 @@
 
 ## Features
 
-### Module
+<div markdown="block" class="alert alert-info">
 
-#### Module add
+**:information_source: Notes about the command format:**<br>
 
-* Create a new module
-* Format: `add module m/<module name>`
+* Lowercase letter(s) followed by a slash are prefixes that prefix a parameter, to be supplied by the user.<br>
+  e.g. in `add module m/<module name>`, `m/` is a prefix that prefixes the `<module name>` parameter.
 
-#### Module add task
+* Words in `<angle brackets>` are the parameters prefixed by prefixes, to be supplied by the user.<br>
+  e.g. in `add module m/<module name>`, `<module name>` is a parameter which can be used as `add module m/CS2105`.
 
-* Create a new task for a specific module
-* Format: `add task m/<module name> a/<task name> d/<task deadline>`
+* Parameters must be prefixed with their specific prefixes.<br>
+  e.g. the command `add module m/CS2105` will add the module CS2105 to TAB, where the commands `add module i/CS2105` and `add module CS2105` will not work.
 
-#### Module add student
+* Parameters must only contain permitted characters, as listed in the [Parameter summary](#parameter-summary-with-prefix-included) below.
 
-* Add a student to a specific module
-* Format: `add student m/<module name> i/<student id> n/<student name> t/<tele_handle> e/<student email>`
+* Optional parameters need not be all present when executing a command, they are denoted in this user guide using square brackets '[]'.<br>
+  e.g. in `edit task m/<module name> ti/<task id> [a/<edited task name>] [d/<edited task deadline>]`, `a/<edited task name>` and `d/<edited task deadline>` are the optional parameters
 
-#### Module delete student
+</div>
 
-* Delete a student from a specific module
-* Format: `module delete <module name>, <student id>`
 
-#### Module delete
+### Add a new module: `add module`
 
-* Delete a module
-* Format: `module delete <module name>`
+Add a new module to TAB.<br>
+Module name must be an NUS module code, consisting of 2-4 capital letters, followed by 4 numbers, followed by 0-4 capital letters.
+Format: `add module m/<module name>`
+
+Examples:
+* `add module m/CS2103`: Add module CS2103 to TAB
```

Comment:
maybe we can standardise and put those after ":" in a new line like the others?

Reply:
agree